converter/marathon.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "/transform",
+  "instances": 1,
+  "cpus": 0.25,
+  "mem": 128,
+  "requirePorts": true,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesosphere/universe-converter:dev"
+    }
+  },
+  "env" : {
+    "MAX_REPO_SIZE" : "20"
+  },
+  "networks" : [
+    {
+      "mode": "HOST"
+    }
+  ],
+  "portDefinitions": [
+    {
+      "name" : "universeconverter",
+      "port" : 8086,
+      "protocol" : "tcp"
+    }
+  ],
+  "labels": {
+    "DCOS_SERVICE_NAME": "transform",
+    "DCOS_SERVICE_PORT_INDEX": "0",
+    "DCOS_SERVICE_SCHEME": "http",
```

Comment:
We shouldn't expose this service through the admin router. Remove all of the `DCOS_SERVICE_*` labels.